srcv2/App.js

```diff
@@ -29,8 +32,17 @@ class App extends React.Component {
                         path='/dashboard'
                         component={DashboardPage}
                     />
```

Comment:
Redirect base url to `DashboardPage` when logged in
```suggestion
                        path={['/dashboard','/']}
                        component={DashboardPage}
                    />
```